app/site/controllers.py

```diff
@@ -27,17 +27,16 @@ def index():
       - site
     responses:
       404:
-        description: Publiser does not exist
+        description: Publisher does not exist
       200:
-        description: Succesfuly loaded home page
+        description: Successfully loaded home page
     """
-    user, exception = get_user_from_cookie()
-    if user:
+    if g.jwt_exception:
+        return redirect(request.headers['Host'] + '/logout')
```

Comment:
Why aren't we using http://flask.pocoo.org/docs/0.12/api/#flask.url_for - and why are we using request.headers stuff?